api/src/resolvers/banking/banking-cypher.ts

```diff
@@ -54,3 +54,26 @@ DELETE r
 
 RETURN record, church.name AS churchName, date.date AS date
 `
+export const lastButOneServiceRecord = `
+MATCH (record:ServiceRecord {id: $serviceRecordId})
+MATCH (record)<-[:HAS_SERVICE]-(:HistoryLog)<-[:HAS_HISTORY]-(fellowship) 
+WHERE fellowship:Fellowship OR fellowship:ClosedFellowship
+WITH fellowship
+MATCH (record:ServiceRecord)<-[:HAS_SERVICE]-(:HistoryLog)<-[:HAS_HISTORY]-(fellowship) 
```

Comment:
This should be :ServiceLog not :HistoryLog

Reply:
Ok. Done